File: leetcode/215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        nums.sort(reverse = True)
        return nums[k-1]
        # [3,2,1,5,6,4]
        #  1,2,3,4,5,6
        # k = 2
        

File: leetcode/215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        multiset<int> setting{};
        for (const auto& x : nums)  {
            setting.insert(x);
            if (setting.size() > k) {
                setting.erase(setting.begin());
            }
        }
        return *setting.begin();
    }
};